documentation.md

```diff
@@ -58,6 +58,7 @@ navigation:
 
 <ul>
   <li><a href="{{site.baseurl}}/docs/2.0.0-preview/">Spark 2.0.0 preview</a></li>
+  <li><a href="{{site.baseurl}}/docs/3.0.0-preview/">Spark 3.0.0 preview</a></li>
```

Comment:
You'll need to restore this by uncommenting it. I guess it's OK to bring back the 2.0.0 preview link as we have every other version here. See comment below about distinguishing preview releases though. Maybe we can just put them above.